Ease.NUnit.Unity/NUnitUnityContainerTestBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Ease.Unity;
+using NUnit.Framework;
+using System;
+
+namespace Ease.NUnit.Unity
+{
+	public abstract class NUnitUnityContainerTestBase : UnityContainerTestBase
+	{
+		private Action _onPerTestSetup;
+
+		public NUnitUnityContainerTestBase()
+		{
+			RegisterPerTestSetup(() => 
+			{
+				_restter.Reset();
```

Comment:
see comments on `DryIocContainerTestBase` and do similar on the Unity side

Reply:
Done